src/lib/dirsize.c

```diff
@@ -147,6 +149,8 @@ double libreport_get_dirsize_find_largest_dir(
         }
  next:
         free(dname);
+        free(sosreport_path);
+        free(lock_path);
```

Comment:
I'd recommend to use `g_autofree` instead. (Thanks, Ernestas.)

Reply:
Right, I actually thought about this right after I pushed the commit. :\

Reply:
Like this? And we got rid of the `goto`, too.

Reply:
Yup, looks better now :+1: